tests/masonry/builders/test_sdist.py

```diff
@@ -191,6 +191,24 @@ def test_find_files_to_add() -> None:
     )
 
 
+def test_find_files_to_add_with_multiple_readme_files() -> None:
+    poetry = Factory().create_poetry(
+        Path(__file__).parent.parent.parent / "fixtures" / "with_readme_files"
+    )
+
+    builder = SdistBuilder(poetry)
+    result = [f.relative_to_source_root() for f in builder.find_files_to_add()]
+
+    assert sorted(result) == sorted(
```

Comment:
Use of sets here would better convey the intentions.

Reply:
This was a no-brainer, I copied from the existing test above :sweat_smile:

Reply:
If you could change the above to use a set comprehension and set literal instead of `sorted(list())` that would be great 😄

Reply:
Great idea, just did :+1: